<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13604" started-at="2020-10-13T10:48:09 IST" finished-at="2020-10-13T10:48:23 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13604" started-at="2020-10-13T10:48:09 IST" finished-at="2020-10-13T10:48:23 IST">
      <class name="com.homeasignment.test.BasicBookingTest1">
        <test-method status="PASS" signature="ReportStartEngine()[pri:0, instance:com.homeasignment.test.BasicBookingTest1@314c508a]" name="ReportStartEngine" is-config="true" duration-ms="17" started-at="2020-10-13T10:48:09 IST" finished-at="2020-10-13T10:48:09 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ReportStartEngine -->
        <test-method status="PASS" signature="scenario()[pri:0, instance:com.homeasignment.test.BasicBookingTest1@314c508a]" name="scenario" duration-ms="13423" started-at="2020-10-13T10:48:09 IST" finished-at="2020-10-13T10:48:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scenario -->
        <test-method status="FAIL" signature="ReportgenerateTestEndEngine(org.testng.ITestResult)[pri:0, instance:com.homeasignment.test.BasicBookingTest1@314c508a]" name="ReportgenerateTestEndEngine" is-config="true" duration-ms="95" started-at="2020-10-13T10:48:23 IST" finished-at="2020-10-13T10:48:23 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=scenario status=SUCCESS method=BasicBookingTest1.scenario()[pri:0, instance:com.homeasignment.test.BasicBookingTest1@314c508a] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[4]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 4
at com.homeasignment.utils.SeleniumDriver.ReportgenerateTestEndEngine(SeleniumDriver.java:57)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:677)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:661)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ReportgenerateTestEndEngine -->
      </class> <!-- com.homeasignment.test.BasicBookingTest1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
